docs/variants/protectli_vp46xx/hardware-matrix.md

```diff
@@ -31,5 +33,33 @@ coreboot port on the Protectli VP46XX firewall.
 | **TPM**                | PC Engines TPM1A LPC TPM                                 |
 | **Power supply**       | Channel Well Technology 12V, 7.5A 90W                    |
 
+## VP4650
+
+| Component              | Description                                              |
+|------------------------|----------------------------------------------------------|
+| **CPU**                | Intel Core i5-10210U                                     |
+| **SSD**                | M.2 SSD: NVMe M.2 2280                                   |
```

Comment:
We should change the approach to test matrixes for devices mounted in the lab. Disks list should correspond to the equipment actually installed.

Reply:
Updated